decisions/handle-database-migrations.md

```diff
@@ -0,0 +1,56 @@
+# Handle database migrations with ...
```

Comment:
@AgustinRamiroDiaz, what do you think about having a number for every ADR so we can know the order in which they were executed? In this case, it would be something like: `decisions/handle-database-migrations.md`

Reply:
I think you meant `decisions/001-handle-database-migrations.md`
sounds good!